src/mumble/UserLocalVolumeSlider.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_USERLOCALVOLUMESLIDER_H_
+#define MUMBLE_MUMBLE_USERLOCALVOLUMESLIDER_H_
+
+
+#include "VolumeSliderWidgetAction.h"
+
+class UserLocalVolumeSlider : public VolumeSliderWidgetAction {
+	Q_OBJECT
+
+	/// The session ID for the user that the dialog is changing the volume for.
+	unsigned int m_clientSession;
+
+public:
+	UserLocalVolumeSlider(QWidget *parent = nullptr);
+	void prepareLocalUser(unsigned int sessionId);
```

Comment:
```suggestion
	void setUser(const ClientUser &user);
```
I think this name would make it a bit clearer what this function is doing :thinking: